src/components/heading/heading.component.js

```diff
@@ -82,15 +92,16 @@ class Heading extends React.Component {
     }
 
     return (
-      <Help
-        className="carbon-heading__help"
-        data-element="help"
-        tooltipAlign="center"
-        tooltipPosition="right"
-        href={this.props.helpLink}
-      >
-        {this.props.help}
-      </Help>
+      <StyledHeaderHelp>
```

Comment:
`StyledHeaderHelp` should be a `styled(Help)` so there's no need for the extra wrapping div here

Reply:
wrapper removed